README.md

```diff
@@ -1,4 +1,9 @@
-# Pine.js
+<div align="center">
+  <img width="400" height="auto" src="https://raw.githubusercontent.com/balena-io/pinejs/master/pinejs.png">
```

Comment:
``` suggestion
<img width="400" height="auto" src="pinejs.png">
```

Reply:
IIRC this won't work when you view the repo README, but I may be wrong...

Reply:
As long as the view is here in GH it should show up well. Is this needed for landr or for local cloned repo view?

Reply:
@fisehara Yeh I believe so, it's the approach I've always taken and I think it's because in some circumstances this path won't work.

Reply:
Okay, fine for me then :)